server/graphql/common/expenses.ts

```diff
@@ -611,6 +609,46 @@ export const markExpenseAsSpam = async (
   return updatedExpense;
 };
 
+const ROLLING_LIMIT_CACHE_VALIDITY = 3600; // 1h in secs for cache to expire
+
+async function validateExpensePayout2FALimit(req, expense, expensePaidAmountKey) {
+  const host = await expense.collective.getHostCollective();
```

Comment:
We can take the host as a parameter and save a SQL query, since all places calling this function already loaded it to check `const hostHasPayoutTwoFactorAuthenticationEnabled = get(host, 'settings.payoutsTwoFactorAuth.enabled', false);`